### PR TITLE
Switch Desktop to 100% TypeScript Loader

### PR DESCRIPTION
**Pull request by rgbkrk**
🎉

### CODE REVIEW

File: applications/desktop/tsconfig.json [hunk 1]
```diff
@@ -1,8 +1,6 @@
 {
   "extends": "../../tsconfig.base.json",
-  "compilerOptions": {
-    "noEmit": true
-  },
+  "compilerOptions": {},
```
Comment (rgbkrk) [inline applications/desktop/tsconfig.json]: Since I had to disable `noEmit`, I wonder if I need to change how reporting works.

Comment (rgbkrk) [reply applications/desktop/tsconfig.json]: Whoops. I sure do, otherwise the type checking generates `.d.ts` files.

Comment (rgbkrk) [reply applications/desktop/tsconfig.json]: Ok, I'm wondering if I should just emit these to an untracked-by-git-location.

**Comment by rgbkrk** [comment]
Alright, this is ready for a looksee.

Pretty happy with `ts-loader` thus far.

**Review by captainsafia — APPROVED** [review]

**Pull request merged**